src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexFiltering.cpp

```diff
@@ -76,27 +99,32 @@ namespace OpenMS
     }
     
     // blacklist low-intensity peaks
-    for (MSExperiment::ConstIterator it_rt = exp_picked_.begin(); it_rt < exp_picked_.end(); ++it_rt)
+    for (MSExperiment::ConstIterator it_rt = exp_centroided_.begin(); it_rt < exp_centroided_.end(); ++it_rt)
     {
       for (MSSpectrum::ConstIterator it_mz = it_rt->begin(); it_mz < it_rt->end(); ++it_mz)
       {
         if (it_mz->getIntensity() < intensity_cutoff_)
         {
-          blacklist_[it_rt - exp_picked_.begin()][it_mz - it_rt->begin()] = 666;
+          blacklist_[it_rt - exp_centroided_.begin()][it_mz - it_rt->begin()] = 666;
         }
       }
     }
     
   }
+  
+  MSExperiment MultiplexFiltering::getCentroidedExperiment()
+  {
+    return(exp_centroided_);
+  }
 
   void MultiplexFiltering::updateWhiteMSExperiment_()
   {
     // reset both the white MS experiment and the corresponding mapping to the complete i.e. original MS experiment
-    exp_picked_white_.clear(true);
-    exp_picked_mapping_.clear();
+    exp_centroided_white_.clear(true);
```

Comment:
comment why in one case true is passed

Reply:
The first one is an `MSExperiment`, the second one is a `vector`.
`true` clears the metadata as well.